blacklight-hierarchy.gemspec

```diff
@@ -28,4 +28,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'engine_cart', '~> 2.3'
   s.add_development_dependency 'capybara'
   s.add_development_dependency 'coveralls'
+  s.add_development_dependency 'net-smtp'
```

Comment:
Huh?

Reply:
I added this to run tests locally, my specific versions of rails and ruby made it necessary. Good call; it should not be included in this PR.